content/tutorial/react/index.md

```diff
@@ -90,11 +130,33 @@ Navigate your Register or Login links or buttons to the following URLs:
 - **LoginRadius App Name** is the name of your app as mentioned in [Get Credential](#get-credentials) step.
 - **return_url** is where you want to redirect users upon successful registration or login. [Whitelist your domain](#whitelist-your-domain) if you are not using Local Domain for this tutorial.
 
+Alternatively, you can use the `loginWithRedirect`, `loginWithPopup` and `logout` authentication methods from the `useLRAuth` hook in your components. We can also access authentication state using `isAuthenticated`. For example: 
+``` JavaScript
+import { useLRAuth } from "loginradius-react";
+
+const example = () => {
+
+const {isAuthenticated,loginWithRedirect,logout } =  useLRAuth();
+   
+  if (isAuthenticated) {
+    return (
+      <div>
+        <button onClick={() => logout({ returnTo: window.location.origin })}>
+          Log out
+        </button>
+      </div>
+    );
+  } else {
+    return <button onClick={loginWithRedirect}>Log in</button>;
+  }
+};       
+``` 
+
 > return_url can be your website, frontend app, or backend server url where you are handling the access token.
 >
 > For this tutorial, set the `return_url` to point to the `/login` subdomain of your application. For example, in the local React instance, it can point to `http://localhost:3000/login`. This way, after logging in through the Auth Page (IDX) Login page, it will redirect the user to the Login component implemented in below [step](#retrieve-user-data-using-access-token).
 
-## Retrieve User Data using Access Token
+## Retrieve User Data 
 
 > Once the authentication is done using Auth Page (IDX), the default script of LoginRadius sends an access token in the query string as a token parameter with the return_url. The return_url should be your application's web page where you would like to receive the access token.
 > The following is an example of the access token in the query string with the Return URL:
```

Comment:
We don't need to use this method right, we can get the accessToken via the `getAccessTokenSilently` method of `useLRAuth`